presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java

```diff
@@ -81,6 +81,7 @@ public void setUp()
         assertQuerySucceeds("CREATE TABLE iceberg.test_schema.iceberg_table1 (_string VARCHAR, _integer INTEGER)");
         assertQuerySucceeds("CREATE TABLE iceberg.test_schema.iceberg_table2 (_double DOUBLE) WITH (partitioning = ARRAY['_double'])");
         assertQuerySucceeds("CREATE TABLE hive.test_schema.hive_table (_double DOUBLE)");
+        assertQuerySucceeds("CREATE VIEW iceberg.test_schema.iceberg_view AS SELECT * FROM iceberg.test_schema.iceberg_table1");
```

Comment:
I see these changes are mostly from TestIcebergMetadataListing.java in https://github.com/trinodb/trino/commit/4b4e4f1c3a161a813757d3c918c371c8f9c48ad1. Can you please squash this commit into "[Add views support in iceberg connector](https://github.com/prestodb/presto/pull/19872/commits/cbd05f2523c630a0ce0a982dde164c89c82a5784)"?

Reply:
Done.